### DISCUSSION TITLE
More icon sizes.

### DISCUSSION THREAD

**Discussion by jdev082** [CatalystDevOrg/Catalyst#114] [General]
@VelocityDesign 
Could you scale the Catalyst icon to the following sizes

24px
32px
64px

So that I can have a clean icon to go with my pinned Catalyst icon on the dock. I define the icon in a desktop file to add it to the app launcher but need a 32px icon to go with that setup. It looks jagged and low res rn since its not properly sized for my dock and launcher.

![image](https://user-images.githubusercontent.com/92550746/147888112-303230b2-3f93-4173-a826-eb01012e3f0c.png)
catalyst icon is at the bottom

**Comment by imdevarsh** [comment]


**Reply by jdev082** [reply]
Taskbars and Docks for Linux Distributions usually use 32px or 64px icons, and very rarely 24px. The reason its so low resolution in my dock because it's downscaling from the giant size the icon is down to 32px

**Reply by jdev082** [reply]
I'm not sure if it supports .ico but im using the png which doesn't scale very well.

**Reply by jdev082** [reply]
So i was able to make the launcher icon use the ico but its still kinda fuzzy 
know this has nothing to do with Catalyst, rather I just need a 32px version of the icon
![image](https://user-images.githubusercontent.com/92550746/147891065-1fe73fb3-c261-44a9-a289-519349c5d4e7.png)

**Reply by imdevarsh** [reply]
Hmm, you can just use an online tool to downscale the png right? Or you can use GIMP.

**Reply by imdevarsh** [reply]
ping

**Reply by jdev082** [reply]
@webdev03 pong

**Reply by imdevarsh** [reply]
Have you got it to work?

**Reply by jdev082** [reply]
@webdev03 why are you asking me to do it? I am nothing but the repository maintainer.

**Reply by imdevarsh** [reply]
I don't use Linux, so I can't test stuff. Please remember that.

**Reply by jdev082** [reply]
@webdev03 that's why you just scale the original icon to 32px and I test it. Since I know how icons are handled I know what will work.

**Reply by imdevarsh** [reply]
Just scale it in GIMP! I don't know how it works on Linux. I have never used it seriously before.

**Reply by jdev082** [reply]
I'm the repository maintainer. I have a big enough job as it is.

**Reply by imdevarsh** [reply]
I'm the repository coder (and collaborator?). I have no idea how to do Linux.

**Reply by jdev082** [reply]
Yeah, you don't need to know anything about linux to scale the ico file to 32px

**Reply by imdevarsh** [reply]
ICO file? What? Isn't that for Windows?

**Reply by jdev082** [reply]
@webdev03 not only

**Reply by imdevarsh** [reply]
@JaydenDev can you do this?

**Reply by jdev082** [reply]
@webdev03 idk why your askin me

**Reply by imdevarsh** [reply]
its your linux system you fix your bugs!

**Reply by jdev082** [reply]
@webdev03 stop spamming useless stuf.

**Comment by icarusinthewings** [comment]
Is this still needed even though development is canceled?

**Reply by jdev082** [reply]
@VelocityDesign it's not canceled anymore...
